crates/uv-build/src/lib.rs

```diff
@@ -808,22 +808,26 @@ impl SourceBuild {
             .join(format!("build_{}.txt", self.build_kind));
 
         debug!(
-            "Calling `{}.build_{}(metadata_directory={})`",
-            pep517_backend.backend, self.build_kind, metadata_directory
+            r#"Calling `{}.build_{}("{}", {}, {})`"#,
+            pep517_backend.backend,
+            self.build_kind,
+            wheel_dir.escape_for_python(),
+            self.config_settings.escape_for_python(),
+            metadata_directory,
         );
         let script = formatdoc! {
             r#"
             {}
 
-            wheel_filename = backend.build_{}("{}", metadata_directory={}, config_settings={})
+            wheel_filename = backend.build_{}("{}", {}, {})
             with open("{}", "w") as fp:
                 fp.write(wheel_filename)
             "#,
             pep517_backend.backend_import(),
             self.build_kind,
             wheel_dir.escape_for_python(),
-            metadata_directory,
             self.config_settings.escape_for_python(),
+            metadata_directory,
```

Comment:
I swapped the order of arguments as `config_settings` is first and `metadata_directory` is second. See: https://peps.python.org/pep-0517/#build-wheel.